modules/file-system/pre-existing-network-storage/README.md

```diff
@@ -67,9 +67,9 @@ No resources.
 |------|-------------|------|---------|:--------:|
 | <a name="input_fs_type"></a> [fs\_type](#input\_fs\_type) | Type of file system to be mounted (e.g., nfs, lustre) | `string` | `"nfs"` | no |
 | <a name="input_local_mount"></a> [local\_mount](#input\_local\_mount) | The mount point where the contents of the device may be accessed after mounting. | `string` | `"/mnt"` | no |
-| <a name="input_mount_options"></a> [mount\_options](#input\_mount\_options) | Options describing various aspects of the file system. | `string` | `""` | no |
-| <a name="input_remote_mount"></a> [remote\_mount](#input\_remote\_mount) | Remote FS name or export (exported directory for nfs, fs name for lustre) | `string` | n/a | yes |
-| <a name="input_server_ip"></a> [server\_ip](#input\_server\_ip) | The device name as supplied to fs-tab, excluding remote fs-name(for nfs, that is the server IP, for lustre <MGS NID>[:<MGS NID>]). | `string` | n/a | yes |
+| <a name="input_mount_options"></a> [mount\_options](#input\_mount\_options) | Options describing various aspects of the file system. Consider adding setting to 'defaults,\_netdev,implicit\_dirs' when using gcsfuse. | `string` | `"defaults,_netdev"` | no |
+| <a name="input_remote_mount"></a> [remote\_mount](#input\_remote\_mount) | Remote FS name or export. This is the exported directory for nfs, fs name for lustre, and bucket name (without gs://) for gcsfuse. | `string` | n/a | yes |
+| <a name="input_server_ip"></a> [server\_ip](#input\_server\_ip) | The device name as supplied to fs-tab, excluding remote fs-name(for nfs, that is the server IP, for lustre <MGS NID>[:<MGS NID>]). This can be omitted for gcsfuse. | `string` | `""` | no |
 
 ## Outputs
```

Comment:
nit: Would you be able to add an example of how to use gcsfuse in the [examples section](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/develop/modules/file-system/pre-existing-network-storage/README.md#example) of `pre-existing-network-storage`.

We don't really want to point people to test_config and without an example it is not super intuitive how to use it.

Reply:
I do think there is enough work on the readme that I would make it a follow-up task.